git-extra/git-sdk.sh

```diff
@@ -20,6 +20,9 @@ case "$rootdir" in
 esac
 
 sdk () {
+	# common options for makepkg calls
+	MAKEPKG_COMMON_OPTS="--syncdeps --noconfirm --skipchecksums --skippgpcheck"
```

Comment:
I am slightly worried that this variable might get re-set between this line and the lines where it is used.

Maybe a better idea would be to introduce a function? I.e. something like

```sh
sdk () {
    case "$1" in
    [...]
    makepkg|makepkg-mingw)
        argv0=/usr/bin/$1
        shift
        exec $argv0 --syncdeps --noconfirm --skipchecksums --skippgpcheck "$@"
        ;;
    [...]
```

Reply:
(And it would be used as e.g. `sdk makepkg --nobuild`.)

Reply:
> Who or what would change the variable's value?

Anyone? The `sdk` function is defined in the scope of the interactive shell. Any `MAKEPKG_COMMON_OPTS=hahaha` in the Git Bash will make this break.

Reply:
I thought the intention of your original change was that users would only use `sdk build` to build stuff. In this case, the variable would be set to the correct value before use. My assumption seems to have been wrong.

Reply:
> I thought the intention of your original change was that users would only use `sdk build` to build stuff.

Yes. And in my mind, the code of `build)` would call the code of `makepkg|makepkg-mingw)`, that way we can avoid relying on a global (and user-visible, and user-modifiable) environment variable.

Reply:
All right. New version incoming.